xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/java/org/xwiki/livedata/LiveDataSelectionConfiguration.java

```diff
@@ -43,11 +46,43 @@ public boolean isEnabled()
 
     /**
      * Sets whether live data entry selection is enabled.
-     * 
+     *
      * @param enabled {@code true} to enable the live data entry selection, {@code false} to disable it
      */
     public void setEnabled(boolean enabled)
     {
         this.enabled = enabled;
     }
+
+    /**
+     * @since 17.4.0RC1
+     */
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        LiveDataSelectionConfiguration that = (LiveDataSelectionConfiguration) o;
+
+        return new EqualsBuilder()
```

Comment:
Not sure it worth creating a EqualsBuilder for a single property. `Objects.equals` should do fine.